cuqi/distribution/_gaussian.py

```diff
@@ -55,10 +84,58 @@ def cov(self):
 
     @cov.setter
     def cov(self, value):
+        if not 'cov' in self._mutable_vars:
+            raise ValueError(f"Mutable variables are {self._mutable_vars}")        
+        value = force_ndarray(value)
+        self._cov = value       
+        if (value is not None) and (not callable(value)):  
+            if self.dim > 75:
```

Comment:
Please define a config variable in `cuqi.config` named e.g. MIN_DIM_SPARSE=75 or something. Then it will be easier to change and update when CUQIpy is running.

Reply:
Okay, I added it to config. How can I then use this in _gaussian.py? I just write the name and that's it? or have to import from config?

Reply:
You have the import cuqi config as any other module

Reply:
I do from cuqi import config, but still MIN_DIM_SPARSE remains undefined. Any suggestion?

Reply:
> I do from cuqi import config, but still MIN_DIM_SPARSE remains undefined. Any suggestion?

``` Python

import cuqi.config

# Where code should be used
if dim > cuqi.config.MIN_DIM_SPARSE

```

Reply:
thanks!